byte DEBUG=1
byte INFO=2
byte WARN=4
byte ERROR=8
byte FATAL=16

string[] paths
byte level
int32 id
uint32 count

---

rosgraph_msgs/Log[] logs
